lib/model/rwfolder.go

```diff
@@ -780,13 +778,9 @@ func (f *rwFolder) renameFile(source, target protocol.FileInfo) {
 	from := filepath.Join(f.dir, source.Name)
 	to := filepath.Join(f.dir, target.Name)
 
-	if f.versioner != nil {
```

Comment:
Maybe I did not made it clear, I'll add a method to each versioner, to restore the old behaviour. And the big change to versioners can be made by someone later.